src/webui/www/private/views/preferences.html

```diff
@@ -7,6 +7,30 @@
         </select>
     </fieldset>
 
+    <fieldset class="settings">
+        <legend>QBT_TR(Action on double-click)QBT_TR[CONTEXT=OptionsDialog]</legend>
+        <table>
+            <tr>
+                <td><label for="dblclick_download_select">QBT_TR(Downloading torrents:)QBT_TR[CONTEXT=OptionsDialog]</label></td>
+                <td>
+                    <select id="dblclick_download_select">
```

Comment:
Please use camel case for `id`.
And please update `dblclick_complete_select` too.

Reply:
Does preference keys need to be changed too?